Automattic/jetpack
# Calypsoify: Add deprecations within Calypsoify file

Fixes https://github.com/Automattic/vulcan/issues/322

## Proposed changes:

* This PR adds deprecations to function and the Jetpack_Calypsoify class, in the Calypsoify module.

### Other information:


## Jetpack product discussion

https://github.com/Automattic/vulcan/issues/322

## Does this pull request change what data or activity we track or use?

No.

## Testing instructions:

* Proof-read. Make sure deprecations are not happening unnecessarily or by default - when checking this branch out and Calypsoifying a post which you can do by visiting the wp-admin post editor for a post and adding `&calypsoify=1` to the end, for example:  `https://example.com/wp-admin/post.php?post=827&action=edit&calypsoify=1.`
* You can also test the deprecations by changing `Calypsoify::get_instance();` to `Jetpack_Calypsoify::get_instance();` in `class-jetpack-calypsoify.php`.
* Additionally, to test individual functions, you can do something similar to the following, also at the bottom of the  file:
```
$instance = Jetpack_Calypsoify::get_instance();
$test_classic_url = $instance->get_switch_to_classic_editor_url();
error_log($test_classic_url );
```
